support-frontend/typescript-errors.json

```diff
@@ -3114,36 +3110,6 @@
             "message": "No overload matches this call."
         }
     ],
-    "TS7016": [
-        {
-            "path": "assets/components/paypalExpressButton/PayPalExpressButton.tsx",
-            "message": "Could not find a declaration file for module 'react-dom'. 'support-frontend/node_modules/react-dom/index.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/components/subscriptionCheckouts/stripeForm/stripeForm.test.tsx",
-            "message": "Could not find a declaration file for module 'react-dom/test-utils'. 'support-frontend/node_modules/react-dom/test-utils.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/helpers/abTests/abtest.ts",
-            "message": "Could not find a declaration file for module 'seedrandom'. 'support-frontend/node_modules/seedrandom/index.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/helpers/rendering/render.ts",
-            "message": "Could not find a declaration file for module 'react-dom'. 'support-frontend/node_modules/react-dom/index.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/helpers/rendering/render.ts",
-            "message": "Could not find a declaration file for module 'preact/debug'. 'support-frontend/node_modules/preact/debug/dist/debug.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/helpers/rendering/ssrPages.ts",
-            "message": "Could not find a declaration file for module 'react-dom/server'. 'support-frontend/node_modules/react-dom/server.js' implicitly has an 'any' type."
-        },
-        {
-            "path": "assets/helpers/tracking/googleTagManager.ts",
-            "message": "Could not find a declaration file for module 'uuid'. 'support-frontend/node_modules/uuid/index.js' implicitly has an 'any' type."
-        }
-    ],
```

Comment:
I think these were caused by generating the error history but missing a few dependencies in node_modules

Reply:
Hi Tom, just to clarify what you mean - would these have not been introduced had someone run yarn before generating the error history?

Reply:
Yes I believe for the most part these would not have been introduced if yarn was run prior to committing